src/App.tsx

```diff
@@ -0,0 +1,22 @@
+import "./App.css";
+import { AppNavBar } from "./components";
+import { Routes, Route } from "react-router-dom";
+import { Home, Quiz } from "./pages";
+
+function App() {
+  return (
+    <div className="App">
+      <AppNavBar />
+
+      <main className="">
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/home" element={<Home />} />
```

Comment:
Why do you require 2 paths for same element to render ? 
When you click on Home tab you can redirect it to / itself

Reply:
did not notice,thanks